src/main/webapp/jsp/instructorFeedbackResultsByQuestion.jsp

```diff
@@ -9,6 +9,6 @@
 <results:resultsPage pageTitle="TEAMMATES - Feedback Session Results" bodyTitle="Session Results" jsIncludes="${jsIncludes}">
     <%-- Responses are displayed when only a single question is loaded, 
          otherwise it is loaded through ajax. See /js/instructorFeedbackResultsAjaxByQuestion.js --%>
-    <results:byQuestionResults isShowingResponses="${!data.largeNumberOfRespondents}"
+    <results:byQuestionResults isShowingResponses="${data.largeNumberOfRespondents}"
```

Comment:
@whipermr5 Yes, you are right. We can be replace that  with `isShowingResponses="${false}"`